packages/neon-domain/__integration__/main.ts

```diff
@@ -0,0 +1,56 @@
+import * as neonCore from "@cityofzion/neon-core";
+import domainPlugin from "../src/index";
+
+const neonJs = domainPlugin(neonCore);
+const { CONST, rpc, u, api, wallet } = neonJs;
+
+let MAINNET_URL = "";
+let provider;
+
+const MAINNET_URLS = [
```

Comment:
Isn't there a full list of mainnet nodes already somewhere inside? this list is very short